src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -62,6 +62,14 @@ def make_artifacts(
 
 @click.command()
 @click.argument("output_dir", type=click.Path(exists=True))
+@click.option(
+    "--version",
+    type=str,
+    default=None,
+    show_default=True,
+    help="Provide a version number for final results. If None is provided, a datetime "
```

Comment:
We aren't generating a datetime if a version is not provided. This help text should be updated.

Reply:
Yeah that was my bad to suggest that. I didn't realize we were going to keep these results in datestamped directories.